save-plugins/fix-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fix/FixPlugin.kt

```diff
@@ -0,0 +1,57 @@
+package org.cqfn.save.plugins.fix
+
+import org.cqfn.save.core.config.SaveConfig
+import org.cqfn.save.core.config.TestSuiteConfig
+import org.cqfn.save.core.files.readLines
+import org.cqfn.save.core.logging.logInfo
+import org.cqfn.save.core.plugin.Plugin
+import org.cqfn.save.core.utils.ProcessBuilder
+
+import io.github.petertrr.diffutils.diff
+import okio.FileSystem
+import okio.Path
+import okio.Path.Companion.toPath
+
+/**
+ * A plugin that runs an executable on a file and compares output with expected output.
+ */
+@Suppress("INLINE_CLASS_CAN_BE_USED")
+class FixPlugin : Plugin {
+    private val pb = ProcessBuilder()
+
+    override fun execute(saveConfig: SaveConfig, testSuiteConfig: TestSuiteConfig) {
+        val fixPluginConfig = testSuiteConfig.pluginConfigs.filterIsInstance<FixPluginConfig>().single()
+        discoverFilePairs(fixPluginConfig.testResources)
+            .also {
+                logInfo("Discovered the following file pairs: $it")
+            }
+            .forEach { (expected, test) ->
+                pb.exec(fixPluginConfig.execCmd.split(" "), null)
+                val fixedLines = FileSystem.SYSTEM.readLines(
```

Comment:
In my case IDE recommends to add annotation ExperimentalFileSystem

Reply:
I've added opt-in as a compiler flag, it should propagate this opt-in for the whole module. I believe you need to reimport the project.